### PR TITLE
Please review2

### PR DESCRIPTION
**Pull request by guilaharis**

### CODE REVIEW

File: arrays.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+
 // Hey!!! PLease review my commit !!!!
 
+
 var favColours = []
 var answer
```
Comment (lilyanavalcheva) [inline arrays.js:7]: please add a word

File: joke [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- What are you?
+- A croissant
+- Oh my God! A croissant that is talking! 
+
```
Comment (lilyanavalcheva) [inline joke:4]: bravo

**Review by lilyanavalcheva — CHANGES_REQUESTED** [review]
hi

File: arrays.js (hunk 1 above)
Comment (guilaharis) [reply arrays.js:7]: ok

**Review by lilyanavalcheva — APPROVED** [review]
bravo

**Pull request merged**